src/parachains.json

```diff
@@ -0,0 +1,14 @@
+[
+"wss://kusama-rpc.polkadot.io",
+"wss://statemine-rpc.polkadot.io",
+"wss://acala-rpc-0.aca-api.network",
+"wss://rpc.parallel.fi",
+"wss://rpc.polkadot.io",
+"wss://node.pol.equilibrium.io",
+"wss://rpc.litentry-parachain.litentry.io",
```

Comment:
shouldn't this be combined with the parachain ids in the other file? Can we just copy the file from Polkadot JS or depend on it?

Reply:
parachains ids is a used here because there is no way to query the parachains with APIto get their id, at the moment. This is mainly used for display in the XCM transfers dashboards, and it can be any parachain, even a one that is not being monitored by the runtime exporter.
But as said, whenever it becomes possible to query all the parachains with their ids with an API, this will be changed.

Reply:
and parachains.json contains the list of parachains to monitor, so they are not the same and there is no reason to merge them